script/Arcueid/Day 13/13_00_ARC13_1.txt

```diff
@@ -1227,7 +1227,8 @@ I call for her to stop. There's one more thing I want to ask her.
 [sha:80869d59b8492d55d7998116113c81f6d1832ff7]{
 -- Page 80, Offset 31902.
 -- 私にとって兄さんは貴方だけです。今も昔も、兄さんは忘れてしまっただろうけど、子供の頃からずっと。
-To me, the word 'brother' refers to you and you alone.
+// Got some feedback from a friend who didn't like this line, so if someone has any ideas, go ahead.
+To me, the word 'brother' refers to you, and you alone.
```

Comment:
A simple, "You are my only brother." maybe?

Reply:
Hmm, I think it's more like "When I hear brother, I only think of you" rather than "there are not two of them". I think 'one and only' maybe brings it around to idiomatic? Some of these suggestions are gonna be lies though.
> You are my one and only brother.
> To me, you are my one and only brother.
> You are the only one I consider my brother.
> To me, you will always be my brother.

Reply:
I guess I like the "You are my one and only brother" out of these

Reply:
Maybe "I don't see anyone but you as my brother" works so she doesn't actually lie?

Reply:
Sure